src/main/resources/XWiki/SuggestDisplay.xml

```diff
@@ -258,13 +264,52 @@
     img.observe('click', this.removeItem.bindAsEventListener(this));
     li.appendChild(span);
     li.appendChild(hiddenInput);
+    //Add the edit button
+    if(this.inEditItemsMode($(this.list).up(0))){
```

Comment:
Maybe you could write:
```
if (this.inEditItemsMode($(this.list))) {
```
or even:
```
if (this.inEditItemsMode()) {
```
so it's the role of `inEditItemsMode()` to get the list item?